.github/workflows/sandpaper-version.txt

```diff
@@ -1 +1 @@
-0.16.5
+0.14.1
```

Comment:
Can we keep the newer version of {sandpaper}?

If anything I think we should upgrade to latest (v 0.16.10)